apps/src/templates/rubrics/EvidenceLevels.jsx

```diff
@@ -12,7 +12,11 @@ export default function EvidenceLevels({
   evidenceLevels,
   canProvideFeedback,
   learningGoalKey,
+  understanding,
+  radioButtonCallback,
 }) {
+  const [understandingChecked, setUnderstandingChecked] =
```

Comment:
What is the difference between this and evaluating `understanding === evidenceLevel.id`?

Reply:
That's a remnant of me trying to get the radio button to display what the user had previously selected if they autosave and return to the page. I'll clean that up.